MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.h

```diff
@@ -31,7 +31,8 @@
 // MarkupsToModel includes
 #include "vtkSlicerMarkupsToModelModuleMRMLExport.h"
 
-static const char* MARKUPS_ROLE = "markupsToModelRoleNode";
+static const char* INPUT_MARKUPS_ROLE = "markupsToModelInputMarkups";
```

Comment:
This role is only used in this class, so no need to prefix it with "markupsToModel". You can just use "InputMarkups" as name.